extensions/commands/recipe/cmd_create_sbom.py

```diff
@@ -0,0 +1,134 @@
+import os.path
+
+from packageurl import PackageURL
+from cyclonedx.factory.license import LicenseFactory
+from cyclonedx.model import ExternalReference, ExternalReferenceType, LicenseChoice, XsUri
+from cyclonedx.model.bom import Bom
+from cyclonedx.model.component import Component, ComponentType
+from cyclonedx.output.json import JsonV1Dot4
+
+from conan.api.output import cli_out_write, ConanOutput
+from conan.api.conan_api import ConanAPI
+from conan.cli.args import common_graph_args, validate_common_graph_args
+from conan.cli.command import conan_command
+from conans.client.graph.graph import Node
```

Comment:
This ``from conans`` import is risky, will break in future releases, I'd avoid it, specially if it is only for types

Reply:
> I'd avoid it, specially if it is only for types

if it was for typing only, then you could follow [pep848 section "Runtime or type checking?"](https://peps.python.org/pep-0484/#runtime-or-type-checking)

Reply:
I am using the import now only during type checking